node/test/util/diff_util.js

```diff
@@ -353,6 +353,9 @@ x=S:C2 a/b=c,a/c=d,t=u;H=2;I a/b,a/q=r,f=x;W a/b=q,a/c=f,a/y=g,f`,
                 input: `
 a=B:Ca-1;Bmaster=a|
 x=S:C2-1 s=Sa:1;Bmaster=2`,
+                staged: {
+                    ".gitmodules": FILESTATUS.ADDED,
```

Comment:
... or added.

Reply:
same as the above, this PR does not add .gitmodules, but it makes changes to .gitmodules visible to git-meta status